content/applications/finance/fiscal_localizations/chile.rst

```diff
@@ -2,898 +2,951 @@
 
-- Taxes.
-- Default Account Payable.
-- Default Account Receivable.
-- Transfer Accounts.
-- Conversion Rate.
+  - :guilabel:`VAT Affected (1st Category)`: used in most cases
+  - :guilabel:`Fees Receipt Issuer (2nd Category)`: for suppliers who issue fees receipt
+  - :guilabel:`End consumer`: only issues receipts
+  - :guilabel:`Foreigner`
 
+- :guilabel:`SII Office`: select your company's :abbr:`SII (Servicio de Impuestos Internos)`
+  regional office
 
-Master Data
------------
+.. _chile/electronic-invoice:
 
-Partner
-~~~~~~~
+Electronic invoice data
+~~~~~~~~~~~~~~~~~~~~~~~
 
-Identification Type and VAT
-***************************
+Select your :guilabel:`SII Web Services` environment:
 
-As part of the Chilean localization, the identification types defined by the SII
-are now available on the Partner form. This information is essential for most transactions.
+- :guilabel:`SII - Test`: for test databases using test :abbr:`CAFs (Folio
+  Authorization Code)` obtained from the :abbr:`SII (Servicio de Impuestos Internos)`. In this mode,
+  the direct connection flows can be tested, with the files being sent to the :abbr:`SII (Servicio
+  de Impuestos Internos)`.
+- :guilabel:`SII - Production`: for production databases.
+- :guilabel:`SII - Demo Mode`: files are created and accepted automatically in demo mode and are
+  **not** sent to the :abbr:`SII (Servicio de Impuestos Internos)`. For this reason, rejection
+  errors or *Accepted with Objections* will not appear in this mode. Every internal validation can
+  be tested in demo mode. Avoid selecting this option in a production database.
 
-.. image:: chile/Chile06.png
-   :align: center
-   :alt: Chilean identification types for partners.
+Then, enter the :guilabel:`Legal Electronic Invoicing Data`:
 
-TaxpayerType
-************
+- :guilabel:`SII Resolution N°`
+- :guilabel:`SII Resolution Date`
 
-In Chile, the document type associated with customers and vendors transactions is defined
-based on the Taxpayer Type. This field should be defined in the partner form, when creating
-a customer is important you make sure this value is set:
+.. image:: chile/electronic-invoice-data.png
+   :alt: Required information for electronic invoice.
 
-.. image:: chile/Chile07.png
-   :align: center
-   :alt: Chilean taxpayer types for partners.
+.. _chile/dte-email:
 
+DTE incoming email server
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Electronic Invoice Data
-***********************
+The :abbr:`DTE (Documentos Tributarios Electrónicos)` :guilabel:`Email Box Electronic Invoicing` can
+be defined to receive your customers' claim and acceptance emails. Enabeling this option is
+necessary if you want to use *Email Box Electronic Invoicing* as the :abbr:`DTE (Documentos
+Tributarios Electrónicos)` incoming email server. To do so, click :guilabel:`Configure DTE Incoming
+Email`. Then, click :guilabel:`New` to add a server and fill in the following fields:
```

Comment:
Just spoke to a CL l10n expert here in SF office. We need this **important** admonition here as we receive a lot of support tickets in relation to this cc: @erla-odoo 
```suggestion
Tributarios Electrónicos)` incoming email server.

.. important::
   In order to receive your SII documents, it's necessary to set up your own email server.
   More information on how to do this can be found in this documentation:
   :doc:`../../general/email_communication/email_servers`

Begin by, clicking :guilabel:`Configure DTE Incoming Email`. Then, click :guilabel:`New` to add a
server and fill in the following fields:
```

Reply:
@StraubCreative the instructions in this section are for setting up an _Incoming Email Server_, I am not sure if linking to the `email_servers.rst` doc is correct since the instructions in the **DTE incoming email server** section are specific to l10n_CL. 

I agree that an **Important** admonition is required to highlight this step, but the instructions are already within this section, no?

cc: @masi-odoo @elre-odoo

Reply:
You could leave it without the link, I suggested having it because some people do want more information regarding setting up the email servers so having the link would lower the amount of tickets submitted for that.

Reply:
@erla-odoo I will leave the link then! I wanted to make sure the link was relevant! 🙂 Thanks!